sources/mysql/infoschema.go

```diff
@@ -207,12 +207,17 @@ func (isi InfoSchemaImpl) GetColumns(conv *internal.Conv, table common.SchemaAnd
 			ignored.AutoIncrement = true
 		}
 		colId := internal.GenerateColumnId()
+		defaultVal := ""
+		if colDefault.Valid {
+			defaultVal=colDefault.String
+		}
```

Comment:
The lint seems to be off, please configure vs code correctly to auto-handle linting.

Reply:
sorry, will correct it.